docker/build.sh

```diff
@@ -156,16 +151,38 @@ if [ "$CREATE_RUNNER" = "YES" ]; then
     fi
 fi
 
+rm -rf ./new_wheelhouse
 if [ "$CREATE_WHL" = "YES" ]; then
     if [ "$BUILD_INHOST" = "YES" ]; then
-        mkdir -p ./wheelhouse
-        cp $(pwd)/${DALI_BUILD_DIR}/nvidia* ./wheelhouse/
+        mkdir -p ./new_wheelhouse
```

Comment:
Why we need this intermidiate `new_wheelhouse` ?

Reply:
done